xen/common/xmalloc_tlsf.c

```diff
@@ -548,6 +549,17 @@ static void *xmalloc_whole_pages(unsigned long size, unsigned long align)
     return res;
 }
 
+void *_xzalloc_whole_pages(unsigned long size, unsigned long align,
```

Comment:
Introduction of an `_xzalloc_whole_pages` should go in a separate patch.

Reply:
will keep it integrated for now until we see this is needed